api/src/client/dynamics/DynamicsLicenseApplicationIdClient.ts

```diff
@@ -38,10 +38,11 @@ export const DynamicsLicenseApplicationIdClient = (
         const mainActiveApplications = response.data.value
           .filter(
             (applicationDetails: LicenseApplicationIdApiResponse) =>
-              applicationDetails.statecode === ACTIVE_STATECODE
+              applicationDetails.statecode !== null && applicationDetails.statecode === ACTIVE_STATECODE
```

Comment:
[pebble] If `applicationDetials.stateCode === ACTIVE_STATECODE`, then it's necessarily not null. What do we gain by adding in a not-null test?

Reply:
I guess it doesn't provide the same value as the other null test as we aren't trying to manipulate the value in anyway, got rid of it.